net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -39,85 +39,50 @@ NO_IPV6=$?
 
 # return true(=0) if has any mwan3 interface enabled
 # otherwise return false
-mwan3_rtmon_ipv4()
+mwan3_rtmon_route()
 {
-	local idx=0
+	config_load mwan3
```

Comment:
We have two config_load mwan3 calls on this line and this https://github.com/openwrt/packages/pull/13041/commits/bbe32d3473207764ea7f6d5c51d0f1024bdfee33

Reply:
Removed one of them here: 3543df28e